source/SymWorld.h

```diff
@@ -1,553 +1,666 @@
+    file.AddHistBin(node, 18, "Hist_0.8", "Count for histogram bin 0.8 to <0.9");
+    file.AddHistBin(node, 19, "Hist_0.9", "Count for histogram bin 0.9 to 1.0");
+
+
+    file.PrintHeaderKeys();
+
+    return file;
+  }
+
+  emp::DataFile & SetupLysisChanceFile(const std::string & filename) {
+    auto & file = SetupFile(filename);
+    auto & node1 = GetSymCountDataNode();
+    auto & node = GetLysisChanceDataNode();
+    node.SetupBins(0.0, 1.1, 11); //Necessary because range exclusive
+    file.AddVar(update, "update", "Update");
+    file.AddMean(node, "mean_lysischance", "Average chance of lysis");
+    file.AddTotal(node1, "count", "Total number of symbionts");
+    file.AddHistBin(node, 0, "Hist_0.0", "Count for histogram bin 0.0 to <0.1");
+    file.AddHistBin(node, 1, "Hist_0.1", "Count for histogram bin 0.1 to <0.2");
+    file.AddHistBin(node, 2, "Hist_0.2", "Count for histogram bin 0.2 to <0.3");
+    file.AddHistBin(node, 3, "Hist_0.3", "Count for histogram bin 0.3 to <0.4");
+    file.AddHistBin(node, 4, "Hist_0.4", "Count for histogram bin 0.4 to <0.5");
+    file.AddHistBin(node, 5, "Hist_0.5", "Count for histogram bin 0.5 to <0.6");
+    file.AddHistBin(node, 6, "Hist_0.6", "Count for histogram bin 0.6 to <0.7");
+    file.AddHistBin(node, 7, "Hist_0.7", "Count for histogram bin 0.7 to <0.8");
+    file.AddHistBin(node, 8, "Hist_0.8", "Count for histogram bin 0.8 to <0.9");
+    file.AddHistBin(node, 9, "Hist_0.9", "Count for histogram bin 0.9 to 1.0");
+
+    file.PrintHeaderKeys();
+
+    return file;
+  }
+
+  emp::DataMonitor<int>& GetHostCountDataNode() {
+    if(!data_node_hostcount) {
+      data_node_hostcount.New();
+      OnUpdate([this](size_t){
+        data_node_hostcount -> Reset();
+        for (size_t i = 0; i< pop.size(); i++)
+          if(IsOccupied(i))
+            data_node_hostcount->AddDatum(1);
+      });
+    }
+    return *data_node_hostcount;
+  }
+
+  emp::DataMonitor<int>& GetSymCountDataNode() {
+    if(!data_node_symcount) {
+      data_node_symcount.New();
+      OnUpdate([this](size_t){
+        data_node_symcount -> Reset();
+        for (size_t i = 0; i < pop.size(); i++){
+        // count both syms in host and potential parallel sym
+          if(IsOccupied(i)){
+            if(pop[i]->IsHost()){
+              data_node_symcount->AddDatum((pop[i]->GetSymbionts()).size());
+            } else data_node_symcount->AddDatum(1);
+          }
+          if (sym_pop[i]){
+            data_node_symcount->AddDatum(1);
+          }
+        }
+      });
+    }
+    return *data_node_symcount;
+  }
+
+  emp::DataMonitor<int>& GetCFUDataNode() {
+    //keep track of host organisms that are uninfected or infected by only lysogenic phage
+    if(!data_node_cfu) {
+      data_node_cfu.New();
+      OnUpdate([this](size_t){
+	  data_node_cfu -> Reset();
+
+	  for (size_t i = 0; i < pop.size(); i++) {
+	    if(IsOccupied(i)) {
+	      if((pop[i]->GetSymbionts()).empty()) {

```

Comment:
You seem to have lost the check for lysogenic, I think you need to redo the merge conflict resolution for this file to make sure you haven't lost anything else important